tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware/algorithm.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, config, engine):
             'CPU': 'AccuracyAwareCommon',
             'VPU': 'AccuracyAwareCommon',
             'GPU': 'AccuracyAwareCommon',
-            'GNA': 'AccuracyAwareGNA'
+            'GNA': 'AccuracyAwareGNA',
```

Comment:
A unit test with the sanity check for GNA2 and GNA3 is required. You can find the baseline in the GNA related tests.

Reply:
I tried to make a unit test with the sanity check in my local, but I couldn't test well since POT ci test is unstable currently.
I'll update the results after the POT CI test is stable and re-request review to you.

Reply:
@bonhunko, please try to use IE_branch param on CI and specify branch based on 1c3f3b4ed073f889bf0417e89883bd435ce4876c commit

Reply:
Thank you for your help daniil!

Reply:
I add a unit test for GNA and GNA3. but it increases the POT CI time by about 1.5-2 hours. @AlexKoff88 Please consider allowing us to omit these unit tests.

Reply:
@bonhunko, such a test should be included in the nightly scope. In pre-commit CI, we check the code is functionally working. It can be done using dummy data (several samples of data, for example, or a small model).

Reply:
I add a sanity check with a synthetic data, please review https://github.com/openvinotoolkit/openvino/pull/12470/commits/89314a2d588bd54e66020983cb20a8870f6cb21b